pilot/pkg/model/jwks_resolver.go

```diff
@@ -107,10 +107,15 @@ type jwksResolver struct {
 	keyChangedCount uint64
 }
 
-// newJwksResolver creates new instance of jwksResolver.
-func newJwksResolver(expireDuration, evictionDuration, refreshInterval time.Duration) *jwksResolver {
-	ret := &jwksResolver{
-		JwksURICache:     cache.NewTTL(jwksURICacheExpiration, jwksURICacheEviction),
+// NewJwksResolver creates new instance of jwksResolver.
```

Comment:
Update comment: jwksResolver -> JwksResolver, and else where.

Reply:
Done!